shared/native/notification-listeners.desktop.js

```diff
@@ -56,8 +54,5 @@ export default function (dispatch: Dispatch, getState: () => Object, notify: any
     'keybase.1.NotifyPGP.pgpKeyInSecretStoreFile': () => {
       dispatch({type: pgpKeyInSecretStoreFile, payload: undefined})
     },
-    'keybase.1.NotifyUsers.userChanged': ({uid}) => {
```

Comment:
there's a TODO to kill this entire thing and replace it with individual actions. this one has been moved to actions/tracker